### PR TITLE
Allows compiling the extra md files with tut

### PR DESCRIPTION
**Pull request by juanpedromoreno**
In addition, it brings the ability to autodeploy snapshots when the branch is master.

Please, @rafaparadela @raulraja could you take a look? Thanks!

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,11 +4,16 @@ scala:
 - 2.10.6
 
 before_install:
- - export PATH=${PATH}:./vendor/bundle
+- export PATH=${PATH}:./vendor/bundle
 
 install:
-  - gem install jekyll -v 2.5
+- gem install jekyll -v 2.5
```
Comment (raulraja) [inline .travis.yml]: Jekyll latest version is [3.2.1](https://rubygems.org/gems/jekyll). IS there a reason why we are using an outdated version?

File: docs/src/main/tut/index.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
+---
```
Comment (raulraja) [inline docs/src/main/tut/index.md]: Has this content been reviewed?

File: docs/src/main/tut/index.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+The plugin provides some basic free styles, css, and image resources by default, everything based [Bootstrap](http://getbootstrap.com/) framework. If you want to personalize color palette, styles and images you can do it in your project side in a easy way as you can see in the [documentation](https://47deg.github.io/sbt-microsites/docs.html).
+
+In order to achieve the microsite creation, it uses directly these plugins and libraries:
+
+* [tut-plugin](https://github.com/tpolecat/tut)
+* [sbt-site](https://github.com/sbt/sbt-site)
+* [sbt-ghpages](https://github.com/sbt/sbt-ghpages)
+* [scalatags](https://github.com/lihaoyi/scalatags)
+
+Additionally it's supported for other useful libraries and plugins like:
```
Comment (raulraja) [inline docs/src/main/tut/index.md]: Additionally it depends on... ?

File: docs/src/main/tut/index.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+First of all, add the following lines to the `project/plugins.sbt` file, within your project or sbt module where you want to use the `sbt-microsites` plugin. Depending on the version:
+
+Latest release:
+```
+addSbtPlugin("com.fortysevendeg"  % "sbt-microsites" % "0.1.0")
+```
+
+Latest snapshot built from the `master` branch code:
+
+```tut:evaluated
```
Comment (raulraja) [inline docs/src/main/tut/index.md]: sweet!

File: docs/src/main/tut/index.md [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+```
+
+Finally, to enable the plugin, add this to your `build.sbt` file:
+```
+enablePlugins(MicrositesPlugin)
+```
+
+## Build the microsite
+
+Once you have written down your documents you can build the microste running this sbt task:
```
Comment (raulraja) [inline docs/src/main/tut/index.md]: microste -> microsite

File: docs/src/main/tut/index.md [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+
+Internally, sequentially it'll run other tasks, among theirs, [`tut`](https://github.com/tpolecat/tut) and `makeSite` ([sbt-site](https://github.com/sbt/sbt-site)) tasks.
+
+## View the microsite in your browser
+
+1. In a shell, navigate to the generated site directory in `target/site`
+
+2. Start jekyll with `jekyll serve`
+
+3. Navigate to http://localhost:4000/yourbase_url/ in your browser, where `yourbase_url` depends on your own preferences (see `micrositeBaseUrl` setting).
```
Comment (raulraja) [inline docs/src/main/tut/index.md]: Is it possible to have jekyll serve this at the root so people just go to localhost for a given site?

File: docs/src/main/tut/index.md (hunk 2 above)
Comment (juanpedromoreno) [inline docs/src/main/tut/index.md]: We should do it, before next release..

File: docs/src/main/tut/index.md (hunk 6 above)
Comment (juanpedromoreno) [inline docs/src/main/tut/index.md]: Yes, it is. `micrositeBaseUrl`is by default empty, so you can see the microsite in `http://localhost:4000` ;)

File: .travis.yml (hunk 1 above)
Comment (juanpedromoreno) [inline .travis.yml]: It's limited by the ruby version installed in Travis. It's just for CI purpose so we shouldn't care.

**Comment by juanpedromoreno** [comment]
Content and lang review will be reviewed as a part of this issue: https://github.com/47deg/sbt-microsites/issues/33

**Pull request merged**